service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcStructuredComponentTest.java

```diff
@@ -41,7 +41,8 @@
 public class GetGpcStructuredComponentTest extends BaseTaskTest {
     private static final String PATIENT_NOT_FOUND = "PATIENT_NOT_FOUND";
     private static final String INVALID_NHS_NUMBER = "INVALID_NHS_NUMBER";
-    public static final List<String> VALID_ERRORS = Arrays.asList(INVALID_NHS_NUMBER, PATIENT_NOT_FOUND);
+    private static final String RCMR_IN030000UK06 = "RCMR_IN030000UK06";
```

Comment:
This constants has the same name as a value. Could we name it differently? For example "EXPECTED_PAYLOAD_TYPE"?